website/docs/components/modal/partials/code/how-to-use.md

```diff
@@ -42,7 +46,9 @@ When a Modal is opened with the keyboard, the focus is automatically set to the
 
 ### Form within a Modal dialog
 
-If a Modal dialog contains interactive elements, such as a form, the initial focus should be set on the first input, which is the first focusable element within the form. This can be achieved by setting the `autofocus` property on the first form element.
+If a Modal dialog presents a form, the initial focus should be set on the first input, as the first focusable element in the form. This can be achieved by setting the `autofocus` property on the first form input.
+
+We recommend placing the `<form>` element in the `Hds::Modal::Body` subcomponent and associating it to the submit button using the `form` attribute, as shown below.
```

Comment:
Is this a recommendation or a "requirement" to work correctly? If the latter, maybe we could use a more direct language, something like this (feel free to edit/adapt)?

```suggestion
The `<form>` element should be placed in the `Hds::Modal::Body` subcomponent. We also recommend to associate it to the submit button using the `form` attribute, as shown below.
```

Reply:
sounds clearer – will update